### PR TITLE
CHEMH-77: A fullpill banner 

### PR DESCRIPTION
**Pull request by jenbreese**
# READY FOR REVIEW

# Summary
- Adding a new banner variant

# Review By (Date)
- 9/22

# Criticality
- Normal

# Urgency
- Normal

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Rebuild Cache and import config `drush cr ; drush ci`
3. Navigate to a page.
4. Add a banner and select the fullpill behavior option
5. The design only has a headline and button with a small image.
6. Verify the banner looks like the design.  The responsive version eventually becomes a circle.  There weren't responsive designs so this might need some changes. 
7. 
image

### Site Configuration Sync

- Is there a config:export in this PR that changes the config sync directory? NO

## Front End Validation
- [ ] Design is approved by @ user?
- [ ] HTML validation: Is the markup using the appropriate semantic tags and [passes validation](https://validator.w3.org/nu/)? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Cross-browser testing: Has been performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Automated accessibility: Scans performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Manual accessibility: Manually tested? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
[- CHEMH-77](https://stanfordits.atlassian.net/browse/CHEMH-77)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

**Review by jdwjdwjdw — CHANGES_REQUESTED** [review]
Looking great overall! Two things:
- Can you address the image getting cut off while stacked on mobile? It looks like that's being caused by the `su-hero__media` aspect ratio
Screenshot 2023-09-19 at 6 04 58 PM
- How do you feel about the image going from a circle to more of an oval right before stacking? I don't mind it too much, but if you think it's an issue, maybe stacking before it goes to the oval would be another possibility
Screenshot 2023-09-19 at 6 05 47 PM

**Comment by jenbreese** [comment]
@jdwjdwjdw I agree with you. I made these changes.  Let me know what you think. Maybe we need to move around the background image to cover the edges better.

File: src/scss/components/banner/_banner--fullpill.scss [hunk 1]
```diff
@@ -0,0 +1,152 @@
+@charset "UTF-8";
+
+$banner-circle--image-width: 200px;
+$banner-circle--sm-image-width: 150px;
+
+
+.chemh-wrapper--banner-fullpill {
+  max-width: calc(100vw - 200px);
```
Comment (jdwjdwjdw) [inline src/scss/components/banner/_banner--fullpill.scss]: It looks like this max-width constraint is causing some issues for mobile, can you address that?
Screenshot 2023-09-21 at 12 33 58 PM

**Review by jdwjdwjdw — CHANGES_REQUESTED** [review]

**Review by jdwjdwjdw — APPROVED** [review]
Everything looking great! 🚀

**Pull request merged**